feat(versioning/debian): Debian codenames support (#14881)
renovatebot/renovate: docs/usage/docker.md

FROM ubuntu:focal
```

### Debian codenames

Renovate understands [Debian release code names and rolling updates schedule](https://wiki.debian.org/DebianReleases) and will offer upgrades to the latest stable release (e.g. from `debian:stretch` to `debian:bullseye`).

> For this to work you must follow this naming scheme:
> 
> - The codename is in lowercase

Reviewer: If all you need to do is to have the codename in lowercase, then we don't need the list format:

```suggestion
For this to work the codename must be in lowercase.
```

For example, Renovate will offer to upgrade the following `Dockerfile` layer:

```dockerfile
FROM debian:buster
```

> To

Reviewer: ```suggestion
To:
```

Let's add punctuation to the end of this very short sentence. 😄
